elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -1387,6 +1455,120 @@ class PfeNavigation extends PFElement {
+    this._searchSpotXs.setAttribute("aria-hidden", "true");
+    this._menuDropdownMd.setAttribute("aria-hidden", "true");
+    this._searchToggle.setAttribute("aria-hidden", "true");
+    this._searchSpotMd.setAttribute("aria-hidden", "true");
+    this._mobileNavSearchSlot.setAttribute("aria-hidden", "true");
+    this._allRedHatToggle.setAttribute("aria-hidden", "true");
+    this._customLinksSlot.setAttribute("aria-hidden", "true");
+
+    this._searchSpotXs.setAttribute("tabindex", "-1");
+    this._menuDropdownMd.setAttribute("tabindex", "-1");
+    this._searchToggle.setAttribute("tabindex", "-1");
+    this._searchSpotMd.setAttribute("tabindex", "-1");
+    this._mobileNavSearchSlot.setAttribute("tabindex", "-1");
+    this._allRedHatToggle.setAttribute("tabindex", "-1");
+    this._customLinksSlot.setAttribute("tabindex", "-1");
+
+    this._searchSpotXs.setAttribute("hidden", "");
+    this._menuDropdownMd.setAttribute("hidden", "");
+    this._searchToggle.setAttribute("hidden", "");
+    this._searchSpotMd.setAttribute("hidden", "");
+    this._mobileNavSearchSlot.setAttribute("hidden", "");
+    this._allRedHatToggle.setAttribute("hidden", "");
+    this._customLinksSlot.setAttribute("hidden", "");
+  }
+
+  /**
+   * Show main menu to screen readers and keyboard users when Back to main menu button is pressed
+   */
+  _showMobileMainMenuSr() {
+    this._searchSpotXs.removeAttribute("aria-hidden", "true");
+    this._menuDropdownMd.removeAttribute("aria-hidden", "true");
+    this._searchToggle.removeAttribute("aria-hidden", "true");
+    this._searchSpotMd.removeAttribute("aria-hidden", "true");
+    this._mobileNavSearchSlot.removeAttribute("aria-hidden", "true");
+    this._allRedHatToggle.removeAttribute("aria-hidden", "true");
+    this._customLinksSlot.removeAttribute("aria-hidden", "true");
+
+    this._searchSpotXs.removeAttribute("tabindex");
+    this._menuDropdownMd.removeAttribute("tabindex");
+    this._searchToggle.removeAttribute("tabindex");
+    this._searchSpotMd.removeAttribute("tabindex");
+    this._mobileNavSearchSlot.removeAttribute("tabindex");
+    this._allRedHatToggle.removeAttribute("tabindex");
+    this._customLinksSlot.removeAttribute("tabindex");
+
+    this._searchSpotXs.removeAttribute("hidden", "");
+    this._menuDropdownMd.removeAttribute("hidden", "");
+    this._searchToggle.removeAttribute("hidden", "");
+    this._searchSpotMd.removeAttribute("hidden", "");
+    this._mobileNavSearchSlot.removeAttribute("hidden", "");
+    this._allRedHatToggle.removeAttribute("hidden", "");
+    this._customLinksSlot.removeAttribute("hidden", "");
+  }
+
+  /**
+   * Set focus to search field when search button is pressed on Desktop
+   * set to the light dom search input field so focus is in the correct place for screen readers and keyboards
+   */
+  _searchFieldFocusHandler() {
+    const searchInput = document.querySelector(".pfe-navigation__search  input");
+    searchInput.focus();
```

Comment:
Should probably wrap this in an `if` statement to make sure it exists, just in case.

Reply:
@wesruv made this change